examples/svelte-kit-demo/svelte.config.cjs

```diff
@@ -5,6 +5,8 @@ module.exports = {
 		// specifying a different adapter
 		adapter: '@sveltejs/adapter-node',
 
+		clientSideRouting: false,
```

Comment:
I wonder if it might be better to make this `routing: { client: false }` or something in case we want to add other routing options. I'm curious what other folks think about names

Reply:
I thought about that maybe there should be an enum for routing, but Rich's original issue had it as `clientSideRouting` so I decided it'd be simpler to leave it as that

Reply:
Aside from the flexibility of adding more routing options, for intellisense purposes it might also be better to provide a `routing` object because then 'client?', 'server?', etc. options would all show up localized in the object.

![image](https://user-images.githubusercontent.com/1460917/111724951-b8193b80-8898-11eb-85e9-20e3383d3b6e.png)